chapter_1/1_is_unique/lib/is_unique.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
```

Comment:
You shouldn't need this string. What happens if you remove it?

Reply:
nothing, that's placed there by rubocop for some reason and I forgot to delete it